pkg/distributor/ha_tracker_test.go

```diff
@@ -897,3 +897,36 @@ func getSumOfHistogramSampleCount(families []*dto.MetricFamily, metricName strin
 
 	return sum
 }
+
+// Test that values are synced to KVStore on starting
```

Comment:
[godocs](https://tip.golang.org/doc/comment) usually start with the name of the function they're describing
```suggestion
// TestStartingSync tests that values are synced to KVStore on startup
```

Reply:
updated on [382baf6](https://github.com/grafana/mimir/pull/5813/commits/382baf63224d637b9bf6ed483113a691d2e97a45)